board/opendingux/package/dingux-commander/dingux-commander.mk

```diff
@@ -0,0 +1,20 @@
+################################################################################
+#
+# DinguxCommander
+#
+################################################################################
+
+DINGUX_COMMANDER_VERSION = stable
```

Comment:
Avoid using a branch here, use a tag or commit ID instead. Otherwise we can't guarantee that two builds from the same commit of the OpenDingux buildroot repo will have the same behaviour.

Reply:
Done